src/components/node-select.vue

```diff
@@ -27,14 +27,31 @@
                 class="status align-self-center"
                 :class="{'icon-connected': isConnected,'icon-disconnected': !isConnected}"
             />
+            <a
+                href="#"
+                class="status align-self-center"
+                @click="logout()"
+            > Log</a>
```

Comment:
Logout

Reply:
that component is not actually used anywhere anymore :)

Reply:
will remove